seq_tools/cli/__init__.py

```diff
@@ -55,10 +56,14 @@ def main(ctx, debug, ignore_update):
 @click.option('--metadata_str', '-s', help='submission metadata as a JSON string')
 @click.option('--data_dir', '-d', type=click.Path(exists=True),
               help='path containing submission data files')
-@click.option('--skip_md5sum_check', is_flag=True, help='skip md5sum check, save time for large files')
+#@click.option('--skip_checks', default=[],nargs=0, help='skip these tests')
+@click.option('--skip_checks','-k', multiple=True,default=[], help='skip this tests')
```

Comment:
Instead of adding the `unskippable` list, why do we just add `type=click.Choice(['c681','c683'], case_sensitive=False)`to only allow these two to be skipped?